website/config/routes.js

```diff
@@ -510,6 +512,7 @@ module.exports.routes = {
   'GET /learn-more-about/chromeos-updates': 'https://support.google.com/chrome/a/answer/6220366',
   'GET /learn-more-about/just-in-time-provisioning': 'https://fleetdm.com/docs/deploy/single-sign-on-sso#just-in-time-jit-user-provisioning',
   'GET /sign-in-to/microsoft-automatic-enrollment-tool': 'https://portal.azure.com',
+  'GET /learn-more-about/custom-os-settings': 'https://fleetdm.com/docs/using-fleet/mdm-custom-os-settings',
```

Comment:
```suggestion
  'GET /learn-more-about/custom-os-settings': '/docs/using-fleet/mdm-custom-os-settings',
```

+ move above next to the others (see also #16270 for what I mean)